src/app/components/search/search-results/search-results.component.ts

```diff
@@ -183,6 +183,10 @@ export class SearchResultsComponent extends PageComponent implements OnInit {
       return null;
     }
 
+    if (userInput.match(/^https?:\/\//)) {
```

Comment:
does this work only for `https` ? what should happen for the `http` input? 
another question: is this code supposed to search documents that are containing hyperlinks?

Reply:
It works for both cases: https and http.  If the document contains url with https and we search url by http, we still get the document.  
This code doesn't allow us to search documents that are containing hyperlinks